reviews/templates/mail/concept_creator.txt

```diff
@@ -15,7 +17,8 @@ Dear colleague,
 
 Your draft application will be checked by your supervisor {{ supervisor }}, the researcher who has the final responsibility of this study.
 You will receive a notification as soon as {{ supervisor }} has approved and formally submitted your draft application, possibly after making some minor revisions.
-In case {{ supervisor }} finds it necessary for you to make more substantial revisions in the application, he/she will notify you about this, and meanwhile formally disapprove the study in order for you to be able to revise and resubmit it.
+
+In case {{ supervisor }} finds it necessary for you to make more substantial revisions in the application, he/she will notify you about this, and meanwhile formally disapprove the application in order for you to be able to revise and resubmit it.
```

Comment:
Paging @miggol 

Desiree wants your feedback on the 'in order for you to be able to revise and resubmit it.' part. Word thinks it's wrong, but doesn't really tell why...

Reply:
I wouldn't call anything here truly incorrect but it is quite hard to read with all these verbs and subclauses.

I would just end with `... in order for you to revise and resubmit it.` which is also less tautological.

Up front you can similarly simplify the sentence by cutting out more verbs, resulting in something like this:

`In case {{ supervisor }} finds more substantial revisions to the application necessary, they will notify you about this, and meanwhile formally disapprove the application in order for you to revise and resubmit it.`

Which at least for me significantly reduces the amount of brainpower required to comprehend the sentence.